src/main/java/org/folio/entlinks/controller/ApiErrorHandler.java

```diff
@@ -63,6 +64,7 @@ public class ApiErrorHandler {
     "authority_source_file_code_unq", DUPLICATE_AUTHORITY_SOURCE_FILE_CODE,
     "pk_authority_storage", DUPLICATE_AUTHORITY_ID,
     "authority_storage_source_file_id_foreign_key", VIOLATION_OF_RELATION_BETWEEN_AUTHORITY_AND_SOURCE_FILE,
+    "authority_archive_source_file_id_foreign_key", VIOLATION_OF_RELATION_BETWEEN_AUTHORITY_ARCHIVE_AND_SOURCE_FILE,
```

Comment:
This map continues to grow. Maybe it's time to move it to some class that will provide an error code based on constraint exception. What do you think?

Reply:
Yeah, it make sense. I'll do some changes.

Reply:
@psmagin please, check my solution. Is it OK I've add default error code to return default value from map?

Reply:
fixed